GPL/vsconfig.gradle

```diff
@@ -32,7 +32,7 @@ def configureVisualStudio() {
 			println "Visual Studio not found!"
 			return
 		}
-		def vswhereOutput = "${vswherePath} -latest -format json".execute().text.trim()
+		def vswhereOutput = "${vswherePath} -products * -latest -requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64 -format json".execute().text.trim()
```

Comment:
Have you experimented with using the `-requires` flag with the other 2 requirements (SDK and ATL)?  If that's possible, maybe it makes sense to do that.